src/index.js

```diff
@@ -11,13 +11,8 @@ import NavBar from './js/navbar'
 import Home from './js/home'
 import AuthPage from './js/authPage'
 import Question from './js/question'
+import NotFound from './js/notfound'
```

Comment:
The naming is Sentence case for the component names and camel case for the file names and other objects